docs/reprocess-report.md

```diff
@@ -0,0 +1,122 @@
+This document describes how to reprocess historical data ingesting new features while minimizing resources waste and negative effect on the on-going data processing.
+
+This document is valid as of March 2019.
```

Comment:
You may want to bump the date up to April 2019.

Reply:
May 2019 also sounds valid :-)